### DISCUSSION TITLE
[Make Bots give a crafting-deconstructing speed boost(in singleplayer?)]

### DISCUSSION THREAD

**Discussion by OnestormGX1** [FakeFishGames/Barotrauma#11775] [Ideas and suggestions]
Add a bonus % overall-multiplicative for every bot in the crew or add a BIG bonus to crafting-deconstructing singleplayer
[Reasoning]: Way easier to code than an entire crafting system for bots, and crafting in singleplayer takes more time than actual playing the rest of the game if you craft the thing yourself (even more if you dont have your crafting stations linked or are new to the game)

**Comment by pantherpusher** [comment]
The task of replacing my oxygen tanks when I can craft max quality tanks is so huge to me and at the same time so compulsively necessary that it causes me to get burned out on campaigns. Combine that with all the other things I should do but never do because they take too long and make me get bored or spend like 45 minutes docked, and suddenly I'm not having fun.
